src/index.html

```diff
@@ -2,13 +2,42 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta 
+      http-equiv="X-UA-Compatible" 
+      content="ie=edge"
+    >
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
```

Comment:
add font from Google fonts to be able to use it on the page
<img width="330" alt="Screenshot 2022-05-03 at 14 14 06" src="https://user-images.githubusercontent.com/57598825/166443475-d5ca1914-636a-4735-ba0b-a6ce78249225.png">